nodeup/pkg/model/kubelet.go

```diff
@@ -145,6 +147,17 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 		flags += " --network-plugin-dir=" + b.CNIBinDir()
 	}
 
+	// If a cluster has a container registry set, flag kubelet to use the pause container from that registry.
+	if b.Cluster.Spec.Assets != nil && b.Cluster.Spec.Assets.ContainerRegistry != nil {
+		// TODO another hardcoded version
```

Comment:
I have been marking all of our hard coded version numbers.  I would like a consolidated place where all of our container versions live.  We have version numbers spread all over our code, which does not seem the best way to organize them.  I would recommend one file for all of our container versions, kinda like urls.go.  Thoughts?